src/cimdemo/EE_geoJSON_ex.py

```
@@ -0,0 +1,181 @@
+        if matched_mapping is not None:
+            return matched_mapping["cimClass"]
+
+    def add_diagram(self):
+        diagram = cim.Diagram(diagram_style=cim.DiagramStyle.GEOGRAPHIC)
+        self.ds.add(diagram)
+        return diagram
+
+    def add_location(self, row):
+        loc = cim.Location()
+        for coord in row["geometry"].coords:
+            logger.info("Creating coordinates: " + coord.__str__())
+            loc.add_point(cim.PositionPoint(coord[0], coord[1]))
+            logger.info('Add Location to Network Service')
+            self.ns.add(loc)
+        return loc
+
+    def add_base_voltages(self):
+        voltages = {'Service': cim.BaseVoltage(nominal_voltage=415, name='415V'),
+                    'LV': cim.BaseVoltage(nominal_voltage=415, name='415V'),
+                    '11kV': cim.BaseVoltage(nominal_voltage=11000, name='11kV'),
+                    'UNKNOWN': cim.BaseVoltage(name='UNKNOWN')}
+        for e in voltages.values():
+            self.ns.add(e)
+        return voltages
+
+    def create_equipment(self, row, loc):
+        class_name = self.get_cim_class(row['class'])
+        if class_name is not None:
+            logger.info("Creating CIM Class: " + class_name)
+            class_ = getattr(cim, class_name)
+            eq = class_()
+            logger.info('Mapping Operating Voltage: ' + self.voltages.get(row['operating voltage'],
+                                                                          self.voltages.get('UNKNOWN')).__str__())
+            logger.info('Creating Equipment:' + ", mRID: " + eq.mrid.__str__())
+            eq.mrid = row["id"]
+            eq.name = row["name"]
+            eq.base_voltage = self.voltages.get(row['operating voltage'])
+            eq.location = loc
+        else:
+            logger.error("GIS Class: " + row['class'] + ", is not mapped to any Evolve Profile class")
```

what does error mean here? you currently report this as an error but continue processing the rest of the feeder. seems like it might make sense to make this fail fast and throw an exception at this point, as if we just log errors we'll either end up missing them and not fixing things, and wasting more time down the track trying to work with broken data.